material-ui 1.0 upgrade

BelfordZ opened the pull request.

BelfordZ committed:
wiwppiwpiwpwpw

shanejonas committed:
fix: update icons for material v1

shanejonas committed:
Solution: icons3 story working with material v1

BelfordZ committed:
remove old icons

shanejonas committed:
prepublish

BelfordZ committed:
first cut of themes

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ETCDEVTeam/emerald-js-ui?pullRequest=54) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/ETCDEVTeam/emerald-js-ui?pullRequest=54) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: BelfordZ<br/>:x: shanejonas<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ETCDEVTeam/emerald-js-ui?pullRequest=54) it.</sub>

BelfordZ committed:
Problem: Address component doesnt work with new material-ui

Solution: Re-add Address component

Problem: Code is not linted with `eslint --fix`

Solution: Generic fixes (linting, fix prop warnings)

Problem: Address component is too complex

Solution: Create new component called ToggledIconButton

BelfordZ committed:
Problem: Material-ui 1.0 has a button component that works for us now.

Solution: Use material-ui button instead of making our own.

BelfordZ committed:
Solution: Remove unneeded components

BelfordZ committed:
remove button group story

BelfordZ committed:
remove link button stuff

BelfordZ committed:
shtuff

BelfordZ committed:
Merge pull request #58 from ETCDEVTeam/remove-junk

Remove junk

BelfordZ committed:
reset lib

BelfordZ committed:
wipwip

BelfordZ committed:
add npm lint rc

BelfordZ committed:
fix eslint up

BelfordZ committed:
run lint fix

BelfordZ committed:
remove color story for now

BelfordZ committed:
remove grabage table

BelfordZ committed:
put back imports

BelfordZ committed:
cleanup

BelfordZ committed:
fixup

BelfordZ committed:
fixups

BelfordZ committed:
prepub

shanejonas committed:
Merge pull request #60 from ETCDEVTeam/material-paper-list

Problem: No list component for emerald styling

shanejonas committed:
fix: toggled icon button classes

shanejonas committed:
initial app bar implementation

shanejonas committed:
feature: continue app bar prototype

shanejonas committed:
feature: green color for checkmark on address copy

shanejonas committed:
fix: account width for shortened

BelfordZ committed:
cleanup cleanup

shanejonas committed:
feature: app bar dropown

BelfordZ committed:
fixups

BelfordZ committed:
fixups

shanejonas committed:
Merge pull request #61 from ETCDEVTeam/cleanup-account

cleanup cleanup

shanejonas committed:
problem: anchor el not working

BelfordZ committed:
fixup

BelfordZ committed:
Merge pull request #62 from ETCDEVTeam/feature/material-ui-one-app-bar

Feature/material ui one app bar

BelfordZ committed:
run prepub

BelfordZ committed:
fix page story

BelfordZ committed:
fix Sync warning component

BelfordZ committed:
prepublish

BelfordZ committed:
add transaction button

BelfordZ committed:
Merge pull request #63 from ETCDEVTeam/tx-button

add transaction button

BelfordZ committed:
prepublish

BelfordZ committed:
add transaction button to .d.ts

BelfordZ committed:
export theme

BelfordZ committed:
export page

BelfordZ committed:
fix typescript defs

BelfordZ committed:
export app bar

BelfordZ committed:
prepublish

BelfordZ committed:
fix

BelfordZ committed:
Declarative components

BelfordZ committed:
wipwip

BelfordZ committed:
wip

BelfordZ committed:
prolly broke everything, enjoy

BelfordZ committed:
go time

BelfordZ committed:
fixme

shanejonas committed:
feature: add CurrentBlockNumber smart component

BelfordZ committed:
remove garbo

BelfordZ committed:
wipwip

shanejonas committed:
feature: add vault rpc

shanejonas committed:
feature: get account selecting working globally

shanejonas committed:
fix: cleanup

shanejonas committed:
feature: pull out global state to story

shanejonas committed:
fix

shanejonas committed:
fixup

shanejonas committed:
fix emerald provider export

shanejonas committed:
feature: add props to emerald provider for default provider urls

BelfordZ committed:
fixups

BelfordZ committed:
Merge pull request #66 from ETCDEVTeam/feature/vault-rpc

Feature/vault rpc

shanejonas committed:
Problem: transaction button doesnt support custom transactions
Solution: use qs to encode transaction object in querystring

shanejonas committed:
Merge pull request #67 from ETCDEVTeam/fix/transaction-button

Problem: transaction button doesnt support custom transactions

shanejonas committed:
feature: add smart contract component

shanejonas committed:
fixup typo

shanejonas committed:
fixup array params

shanejonas committed:
fix: props passing and decoding

shanejonas committed:
fix: render props

shanejonas committed:
fix: add refresh call to smart contract

shanejonas committed:
fix: get refresh working on ethrpc

shanejonas committed:
fix: use willunmount

shanejonas committed:
fix: remove context from render prop fetchers

shanejonas committed:
fix: set data on component did mount for contract

BelfordZ committed:
fix package lock and add error if method not defined

shanejonas committed:
fix package-lock.json

shanejonas committed:
fix package lock again

BelfordZ committed:
remove material-ui for real

BelfordZ committed:
prepublish

BelfordZ committed:
Merge pull request #69 from ETCDEVTeam/sc-material-ui-remove

Sc material ui remove

BelfordZ committed:
fixup

BelfordZ committed:
prepublish

shanejonas committed:
feature: add transaction uri

shanejonas committed:
Merge branch 'feature/smart-contract' of github.com:ETCDEVTeam/emerald-js-ui into feature/smart-contract

shanejonas committed:
fix npm prepublish

shanejonas committed:
fix setdata again

shanejonas committed:
feature: theme cleanup

shanejonas committed:
fix: list and gutter styling

shanejonas committed:
feature: add bigger gutters and muilist theme

shanejonas committed:
fix: more styles + add jsx addon

shanejonas committed:
fix: padding on muilist

shanejonas committed:
fix: margin bottom for typography theme

shanejonas committed:
fix: network selector and current block number to use theme

shanejonas committed:
Merge pull request #70 from ETCDEVTeam/feature/theme-cleanup

Feature/theme cleanup

BelfordZ committed:
Merge pull request #68 from ETCDEVTeam/feature/smart-contract

Feature/smart contract

BelfordZ committed:
Merge pull request #65 from ETCDEVTeam/declerative-material-one

Declarative components

BelfordZ committed:
build storybook

shanejonas committed:
fix: orphaned components in ts file

BelfordZ commented:
lol rip
